src/dlt-qnx-system/dlt-qnx-slogger2-adapter.cpp

```diff
@@ -162,11 +165,12 @@ static void wait_for_buffer_space(const double max_usage_threshold,
         timeout = std::chrono::steady_clock::now() < end_time;
```

Comment:
```suggestion
        timeout = std::chrono::steady_clock::now() >= end_time;
```

Reply:
Hi @michael-methner
I tested your PR on a QNX system and yet Slog is spammed with "failed to get enough buffer space" msg. I think the timeout condition is incorrect.